view/frontend/web/css/unzer.css

```diff
@@ -0,0 +1,5206 @@
+.unzerUI.divider {
```

Comment:
Did you download the current one and adapted it?
Is there a way to include the one from static.unzer.com and just override the changes?

Reply:
I am not completly sure how you mean this. I have overritten content and renamed it.

Reply:
ok fine